mParticle-Apple-SDK/Custom Modules/MPCustomModulePreference.m

```diff
@@ -135,10 +135,10 @@ - (NSString *)defaultValueForMacroPlaceholder:(NSString *)macroPlaceholder __att
         CFUUIDRef uuidRef = CFUUIDCreate(kCFAllocatorDefault);
         CFUUIDBytes uuidBytes = CFUUIDGetUUIDBytes(uuidRef);
         
-        SInt64 lsbBytes[8] = {uuidBytes.byte8, uuidBytes.byte9, uuidBytes.byte10, uuidBytes.byte11,
+        uint64_t lsbBytes[8] = {uuidBytes.byte8, uuidBytes.byte9, uuidBytes.byte10, uuidBytes.byte11,
                               uuidBytes.byte12, uuidBytes.byte13, uuidBytes.byte14, uuidBytes.byte15};
         
-        SInt64 value = 0;
+        uint64_t value = 0;
```

Comment:
Correct me if I'm wrong, but I think this going to change behavior if you don't cast back into a signed integer afterwards.